src/main/javascript/crypto/src/eas/EasTicketAttestation.ts

```diff
@@ -350,14 +368,14 @@ export class EasTicketAttestation extends AttestableObject implements Attestable
 				this.signedAttestation.domain.verifyingContract,
 				this.signedAttestation.domain.chainId
 			]
-		)
+		);
 
 		const attestation = defaultAbiCoder.encode(
 			this.signedAttestation.types.Attest.map((field) => field.type),
 			this.signedAttestation.types.Attest.map((field) => this.signedAttestation.message[field.name])
 		);
 
-		const signature = joinSignature(this.signedAttestation.signature);
+		const signature = joinSignature(this.signedAttestation.signature ?? this.signedAttestation);
```

Comment:
Hi, @micwallace , could you explain this change? 
this.signedAttestation is SignedOffchainAttestation but joinSignature need SignatureLike param

Reply:
In a previous version of the signed attestation JSON, the r,s,v parameters were in their own signature property (making it compatible with SignatureLike type). Later they were moved into the top level object. This change is just for backward compatibility.